src/safety/SafetyConfig.js

```diff
@@ -74,27 +75,22 @@ const safetyConfig = {
       legendOptions: {
         title: "Road risk score"
       },
-      classBreakInfos: [{
-        minValue: 0,
-        maxValue: 0.5,
-        symbol: {type: "simple-line", width: "2.5px", color: [171,217,233,255], onHoverScale},
-        label: "Low"
-      }, {
-        minValue: 0.5,
-        maxValue: 1,
-        symbol: {type: "simple-line", width: "2.5px", color: [255,255,191,255], onHoverScale},
-        label: "Average"
-      }, {
-        minValue: 1,
-        maxValue: 2,
-        symbol: {type: "simple-line", width: "2.5px", color: [253,174,97,255], onHoverScale},
-        label: "High"
-      }, {
-        minValue: 2,
-        maxValue: 1000,
-        symbol: {type: "simple-line", width: "2.5px", color: [215,25,28,255], onHoverScale},
-        label: "Very High"
-      }]
+      classBreakInfos: getClassBreakInfos({
+        stops: [0, 0.5, 1, 2, 1000],
+        labels: ["Low", "Average", "High", "Very High"],
+        colors: [[171,217,233,1], [255,255,191,1], [253,174,97,1], [215,25,28,1]],
+        type: "simple-line", 
+        width: "2.5px"
+      }),
+      classBreakInfosByQuery: {
+        "project <> 'me8'": getClassBreakInfos({
+          stops: [0, 0.2, 0.75, 1.5, 500],
+          labels: ["Low", "Average", "High", "Very High"],
```

Comment:
Maybe it's better to create a second set of renderers than build additional layer of complexity...